cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,50 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  // before(() => {
+  //   cy.visit('/login')
+  // });
+  const user = {
+    username: 'tester_1',
+    password: 'Tester1!'
+  };
 
+  const book = {
+    isbn: '9781449365035',
+    userId: 'a66cc17b-e2fc-40a5-920c-f6a9115834f9',
+    title: 'Speaking JavaScript'
+  };
+
+  it('should provide an ability to log in', () => {
+    cy.visit('/login');
```

Comment:
We can use our `login()` function here too!

If you want to do it suuuper well, add logging into the `beforeEach()` hook. You won't have to add it to each test separately!

Reply:
fixed